bootstrap/kubeadm/api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -18,7 +18,7 @@ package v1alpha2
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kubeadmv1beta1 "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/kubeadm/v1beta1"
+	kubeadmv1beta1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/kubeadm/v1beta1"
```

Comment:
Maybe in a follow-up we can find an alternative to kubeadm/kubeadm?

Reply:
yes. This bugged me when I made the change but it's something we can address later.

Reply:
Follow up issue: https://github.com/kubernetes-sigs/cluster-api/issues/1544